packages/asc-core/src/Radio/Radio.tsx

```diff
@@ -60,7 +62,7 @@ const Wrapper = styled.div`
 `
 
 const InputRadio = (props: any) => (
```

Comment:
Like discussed, we don't use react in the asc-core package 👿 😉 . Consider moving this to the asc-ui?